### PR TITLE
ref #218 update tutteli-spek-extensions to 1.0.1

### PR DESCRIPTION
**Pull request by ShradhaSangtani**
----
I confirm that I have read the [Contributor Agreements v1.0](https://github.com/robstoll/atrium/blob/master/.github/Contributor%20Agreements%20v1.0.txt), agree to be bound on them and confirm that my contribution is compliant.

### CODE REVIEW

File: apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/PathAssertionsSpec.kt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package ch.tutteli.atrium.api.fluent.en_GB.jdk8
 
 import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.creating.PathAssertionsBuilder.exists
```
Comment (robstoll) [inline apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/PathAssertionsSpec.kt]: please revert

File: misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathFeatureAssertionsSpec.kt [hunk 2]
```diff
@@ -51,12 +52,12 @@ abstract class PathFeatureAssertionsSpec(
             it("toBe(folder.parent) holds") {
                 val childFolder = tempFolder.newFolder("child")
                 val parentFolder = childFolder.parent
-                expect(childFolder).parentVal().toBe(parentFolder)
+                toBe(parentFolder,expect(childFolder))
```
Comment (robstoll) [inline misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathFeatureAssertionsSpec.kt]: don't use `AnyAssertionsBuilder.toBe` it was perfectly fine how it was previously. I guess there is a misunderstanding in the goal of the task. You need to change how `tempFolder` is specified and used.

**Comment by ShradhaSangtani** [comment]
@robstoll Getting the error -  **Cannot add extension with name 'kotlin', as there is an extension already registered with that name.** when i try to run the updated project. Can you help me with this?

File: misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathFeatureAssertionsSpec.kt (hunk 2 above)
Comment (ShradhaSangtani) [reply misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathFeatureAssertionsSpec.kt]: So i was getting errors on both tempFolder and toBe. For tempFolder i could see there is another class MemoizedTempFolder. But it does not have a method perTest() or anything similar to that.

Comment (robstoll) [reply misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathFeatureAssertionsSpec.kt]: That's correct, there isn't a perTest or perGroup anymore, this is control via CachingMode of memoized. See https://github.com/robstoll/tutteli-spek-extensions#MemoizedTempFolder for an example how it is used.

**Comment by robstoll** [comment]
Thanks for pointing out the error, that's a bug on master. I have fixed it. Please rebase

**Pull request closed without merging**